test/e2e/kubectl.go

```diff
@@ -97,11 +97,11 @@ var _ = Describe("kubectl", func() {
 	Describe("guestbook", func() {
 		var guestbookPath = filepath.Join(testContext.RepoRoot, "examples/guestbook")
 
-		if testContext.Provider != "gce" && testContext.Provider != "gke" {
-			By(fmt.Sprintf("Skipping guestbook, uses createExternalLoadBalancer, a (gce|gke) feature"))
-		}
-
 		It("should create and stop a working application", func() {
+			if testContext.Provider != "gce" && testContext.Provider != "gke" {
+				By(fmt.Sprintf("Skipping guestbook, uses createExternalLoadBalancer, a (gce|gke) feature"))
```

Comment:
Lowercase is used for error strings, but not log statements. See https://github.com/golang/go/wiki/CodeReviewComments#error-strings which says "This does not apply to logging, which is implicitly line-oriented and not combined inside other messages." and By() statements are logging statements.